File: bwl.Imaging/Bitmap/TextObject.vb

Imports System.Drawing

Public Class TextObject

    Public Property Point1 As PointF
    Public Property Text As String = ""
    Public Property Size As Single = 0.03

    Public Sub New()
    End Sub

    Public Sub New(point1 As PointF, text As String, size As Single)
        Me.Text = text
        Me.Point1 = point1
        Me.Size = size
    End Sub

    Public Sub New(x1 As Single, y1 As Single, text As String, size As Single)
        Me.Point1 = New PointF(x1, y1)
        Me.Text = text
        Me.Size = size
    End Sub
End Class

File: Bwl.Imaging/Bitmap/TextObject.cs

using System.Drawing;

namespace Bwl.Imaging
{

    public class TextObject
    {

        public PointF Point1 { get; set; }
        public string Text { get; set; } = "";
        public float Size { get; set; } = 0.03f;

        public TextObject()
        {
        }

        public TextObject(PointF point1, string text, float size)
        {
            Text = text;
            Point1 = point1;
            Size = size;
        }

        public TextObject(float x1, float y1, string text, float size)
        {
            Point1 = new PointF(x1, y1);
            Text = text;
            Size = size;
        }
    }
}